packages/video_player/video_player/test/web_vtt.test.dart

```diff
@@ -0,0 +1,176 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
```

Comment:
usually, we name the testing files as words separated with `_`, so `web_vtt_test.dart`